src/Types.hs

```diff
@@ -268,6 +268,7 @@ data Config = Config
   , configSmartBacktick             :: Bool
   , configURLOpenCommand            :: Maybe Text
   , configURLOpenCommandInteractive :: Bool
+  , configActivityNotifyCmd         :: Maybe T.Text
```

Comment:
On second thought, please expand `Cmd` to `Command` here, too, just as with the setting name, to be consistent with the other settings that hold commands.